pkg/controllers/cloud/account_poller.go

```diff
@@ -346,6 +372,52 @@ func updateAccountStatus(current, discovered *cloudv1alpha1.CloudProviderAccount
 	current.Error = discovered.Error
 }
 
+func addAccountPoller(cloudType cloudv1alpha1.CloudProvider, namespacedName *types.NamespacedName,
```

Comment:
Why arent they part of accountPoller interface?? Please avoid declaring function like this.

Reply:
It can't be a part of account poller interface, here we creating the account poller. We can tie it to CPA reconciler(which i did in earlier patches) as CPA creates/deletes account poller. If we do that we need to store CPA reconciler pointer in CES reconciler as CES also needs to update account poller object once CES is added. selector and vmSelector fields in accountPoller struct are populated by CES. 
This is the reason why it was written like this.

Reply:
Another way is to create a new struct for Poller which holds accPoller map(instead of keeping it global), something like
type Poller struct {
	accPollers     map[types.NamespacedName]*accountPoller
	mutex sync.Mutex
	log            logr.Logger
}

func InitPollers() *Poller {
	poller := &Poller{
		accPollers: make(map[types.NamespacedName]*accountPoller),
		log:        logging.GetLogger("Poller").WithName("AccountPoller"),
	}
	return poller
}

Now InitPollers() can invoked from main.go and later pass this poller pointer to CES, CPA reconcilers as they need to operate on accPoller objects.